login.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+require_once 'connectToDB.php';
+require_once 'src/Models/UsersModel.php';
+require_once 'SessionHandler.php';
+
+session_start();
+
+$session = new SessionHandles();
+if ($session->checkUserLoggedIn()) {
+    header('Location: index.php');
+}
+
+
+if (isset($_POST['email'], $_POST['password'])) {
+
+    $inputtedEmail = $_POST['email'];
+    $inputtedPassword = $_POST['password'];
+
+    $db = connectToDB();
+    $usersModel = new UsersModel($db);
+
+    $users = $usersModel->selectUser($inputtedEmail);
+    if ($users === null) {
+        echo 'User does not exist';
```

Comment:
weird place to echo. better to put this in a function, return this strong then run the function in the HTML where you actually want the output